src/main/kotlin/io/github/yuutoproject/yuutobot/commands/minigame/MinigameInstance.kt

```diff
@@ -0,0 +1,182 @@
+                    "Please run `minigame skip` in order to skip a question."
+            )
+        val startingMessage = event.channel.sendMessage(embed.build()).complete()
+        startingMessageID = startingMessage.id
+
+        startingMessage.addReaction("U+1F1F4").complete()
+
+        for (countdown in 10 downTo 0 step 2) {
+            embed.setDescription(
+                "React below to join the game!\n" +
+                    "This game may contain spoilers or NSFW themes.\n" +
+                    "Please run `minigame skip` in order to skip a question.\n" +
+                    "Current players: ${getPlayers()}\n" +
+                    "$countdown seconds left!"
+            )
+            startingMessage.editMessage(embed.build()).complete()
+
+            Thread.sleep(2000)
+        }
+
+        if (players.isEmpty()) {
+            embed.setTitle("Minigame cancelled!").setDescription("Nobody joined...")
+            startingMessage.editMessage(embed.build()).complete()
+            manager.unregister(this)
+            return
+        }
+
+        embed.setTitle("Minigame started!").setDescription("Game has begun!")
+        startingMessage.editMessage(embed.build()).complete()
+
+        begun = true
+        progress(event)
+    }
+
+    fun progress(event: GuildMessageReceivedEvent) {
+        if (rounds > maxRounds) {
+            endGame(event)
+            return
+        }
+
+        // Unfortunately, no removeOrNull, so we have to use a try/catch
+        try {
+            currentQuestion = questions.removeAt(0)
+        } catch (e: IndexOutOfBoundsException) {
+            endGame(event)
+            return
+        }
+
+        currentAnswers = currentQuestion.answers.map { it.toLowerCase() }
+            .toMutableList()
+
+        if (currentQuestion.type == "FILL") {
+            event.channel.sendMessage(currentQuestion.question).queue()
+        } else if (currentQuestion.type == "MULTIPLE") {
+            val questionString = "${currentQuestion.question}\n"
+
+            val answerString = (currentQuestion.wrong + currentQuestion.answers).shuffled()
+                .mapIndexed { i, answer ->
+                    if (currentAnswers.contains(answer.toLowerCase())) {
+                        currentAnswers.add((i + 1).toString())
+                    }
+
+                    "${i + 1}) $answer"
+                }.joinToString("\n")
+
+            event.channel.sendMessage(questionString + answerString).queue()
+        }
+    }
+
+    private fun endGame(event: GuildMessageReceivedEvent) {
+        // Sort by descending value and then map each value to a line in the scoreboard, then join it
+        val scoreboard = players.entries.sortedByDescending { it.value }.mapIndexed { i, entry ->
+            "${i + 1}) <@${entry.key}> with ${entry.value} points"
+        }.joinToString("\n")
+
+        val embed = EmbedBuilder()
+            .setColor(Color.BLUE)
```

Comment:
```suggestion
            .setColor(0xFF93CE)
```
Use Yuuto Pink in all Embeds.